uuid-kinds/src/lib.rs

```diff
@@ -2,26 +2,26 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-#![cfg_attr(not(feature = "std"), no_std)]
-
 //! A registry for UUID kinds used in Omicron and related projects.
 //!
 //! See this crate's `README.adoc` for more information.
 
+#![cfg_attr(not(feature = "std"), no_std)]
+
 // Export these types so that other users don't have to pull in newtype-uuid.
 #[doc(no_inline)]
 pub use newtype_uuid::{
     GenericUuid, ParseError, TagError, TypedUuid, TypedUuidKind, TypedUuidTag,
 };
 
-#[cfg(feature = "schemars08")]
+#[cfg(all(feature = "schemars08", feature = "std"))]
```

Comment:
@sunshowers this is what I referenced in matrix.

Reply:
schemars requires std, so I'd rather the schemars08 feature depend on std in `Cargo.toml` -- what do you think?